addons/mass_mailing/models/mailing.py

```diff
@@ -199,7 +199,7 @@ def _compute_email_from(self):
     def _compute_total(self):
         for mass_mailing in self:
             total = self.env[mass_mailing.mailing_model_real].search_count(mass_mailing._parse_mailing_domain())
-            if total and mass_mailing.ab_testing_pc < 100:
+            if total and mass_mailing.ab_testing_enabled:
```

Comment:
(Can't we keep the "< 100" part though?)

Reply:
Yeah sure, it's just not particularly necessary I guess.
Will put it back!